eve/main.yml

```diff
@@ -87,3 +88,25 @@ stages:
           kubectl get pv
           ! kubectl get pv -o jsonpath={.items[*].status.phase}| grep 'Released' > /dev/null
         env: *env_kubeconfig
+
+  build-doc-inter-stage:
+    worker:
+      type: kube_pod
+      path: eve/workers/zenko.yaml
+    steps:
+    - TriggerStages:
+        name: Stage to defer the build of docker of doc-builder
+        stage_names:
+        - build-doc
+
+  build-doc:
+    worker:
+      type: kube_pod
+      path: eve/workers/doc-builder.yaml
+      images:
+        doc-builder: docs/
+    steps:
+    - Git: *git_pull
+    - ShellCommand:
+        name: 'Build doc'
+        command: make -C docs html
```

Comment:
This is not the same as what the `tox` target runs: as an example, this will not (though should!) bail out when Sphinx generates warnings.
We'll also want to run the `spelling` target (or however it's called).

Reply:
Might be better to run tox directly. But I don't know if it's possible to download dependencies in dockerfile and then tell tox to use what's in the docker image

Reply:
I believe the following in the `Dockerfile` should make `tox` generate the environment (i.e. including installation of the dependencies etc) without running the *test command* (which is in this case, building the docs):

```shell
RUN mkdir docs
COPY tox.ini .
COPY docs/requirements.txt docs/
RUN tox -e docs --notest

CMD tox -e docs
```